extensions/interactions/MusicNotesInput/directives/oppia-interactive-music-notes-input.directive.ts

```diff
@@ -189,7 +189,6 @@ angular.module('oppia').directive('oppiaInteractiveMusicNotesInput', [
         // Highest number of notes that can fit on the staff at any given time.
         var MAXIMUM_NOTES_POSSIBLE = 8;
 
-        var noteChoicesElt = element.find('.oppia-music-input-note-choices');
```

Comment:
Just to check, have you verified that this interaction still works correctly, since there seem to be a lot of changes here?

Reply:
Fixed it and other files

Reply:
I did not mean for you to revert these changes. Having an element.find() without any variable doesn't seem useful at all.
I was just asking with your previous version (removing these  lines) that the interaction was working as expected.

Reply:
This interaction is working the same as before removing the unused vars